### PR TITLE
{math}[GCC/11.3.0] SVG v2.87, FLINT v2.9.0, polymake v4.8

### PR DESCRIPTION
**Pull request by terjekv**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by terjekv** [comment]
Test report by @terjekv
**SUCCESS**
Build succeeded for 0 out of 0 (3 easyconfigs in total)
ninhursaga.uio.no - Linux RHEL 9.1 (Plow), x86_64, Intel(R) Xeon(R) Silver 4216 CPU @ 2.10GHz, 1 x NVIDIA Tesla T4, 525.60.13, Python 3.9.14
See https://gist.github.com/e09383ac8b86613745587a43c9d68968 for a full test report.

File: easybuild/easyconfigs/p/polymake/polymake-4.8-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    ('libxml2', '2.9.13'),
+    ('libxslt', '1.1.34'),
+    ('libreadline', '8.1'),
+    ('Perl', '5.34.1'),
+    ('SVG', '2.87'),
+    ('GMP', '6.2.1'),
+    ('MPFR', '4.1.0'),
+    ('Boost', '1.79.0'),
+    ('FLINT', '2.9.0'),
+    ('Java', '15', '', True),
```
[the remark this answers is not in the document]
Comment (terjekv) [reply easybuild/easyconfigs/p/polymake/polymake-4.8-foss-2022a.eb:28]: ... Thanks, I managed to think 15 was default in 2022a. No idea how that happened. Updating the PR.

[the remark this answers is not in the document]
Comment (jfgrimm) [reply easybuild/easyconfigs/p/polymake/polymake-4.8-foss-2022a.eb:28]: np, I think we considered going with something newer than 11 but it's not an LTS

**Comment by terjekv** [comment]
Test report by @terjekv
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
ninhursaga.uio.no - Linux RHEL 9.1 (Plow), x86_64, Intel(R) Xeon(R) Silver 4216 CPU @ 2.10GHz, 1 x NVIDIA Tesla T4, 525.60.13, Python 3.9.14
See https://gist.github.com/8106ea86984b0a6554fe9ab4e3605058 for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 4 out of 4 (3 easyconfigs in total)
node087.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/a13cc4f0b8af525f0af262c7898960fe for a full test report.

File: easybuild/easyconfigs/p/polymake/polymake-4.8-foss-2022a.eb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+source_urls = ['https://polymake.org/lib/exe/fetch.php/download/']
+sources = [SOURCE_TAR_BZ2]
+checksums = ['3f7bd19d6ac1372ae32368e57134ff37a11fbe9eb1f6abe8195b34c5eec29b11']
+
+dependencies = [
+    ('Ninja', '1.10.2'),
+    ('libxml2', '2.9.13'),
+    ('libxslt', '1.1.34'),
+    ('libreadline', '8.1'),
```
Comment (jfgrimm) [inline easybuild/easyconfigs/p/polymake/polymake-4.8-foss-2022a.eb:21]:
```suggestion
    ('libreadline', '8.1.2'),
```

File: easybuild/easyconfigs/p/polymake/polymake-4.8-foss-2022a.eb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    ('Perl', '5.34.1'),
+    ('SVG', '2.87'),
+    ('GMP', '6.2.1'),
+    ('MPFR', '4.1.0'),
+    ('Boost', '1.79.0'),
+    ('FLINT', '2.9.0'),
+    ('Java', '11', '', True),
```
Comment (jfgrimm) [inline easybuild/easyconfigs/p/polymake/polymake-4.8-foss-2022a.eb:28]:
```suggestion
    ('Java', '11', '', SYSTEM),
```

File: easybuild/easyconfigs/s/SVG/SVG-2.87-foss-2022a.eb [hunk 4]
```diff
@@ -0,0 +1,26 @@
+easyblock = 'PerlModule'
+
+name = 'SVG'
+version = '2.87'
+
+homepage = 'https://metacpan.org/pod/SVG'
+description = "Perl binding for SVG"
+
+toolchain = {'name': 'foss', 'version': '2022a'}
```
Comment (jfgrimm) [inline easybuild/easyconfigs/s/SVG/SVG-2.87-foss-2022a.eb:9]: I would think that `GCCcore` would be sufficient?
```suggestion
toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
```

**Review by jfgrimm — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/p/polymake/polymake-4.8-foss-2022a.eb (hunk 2 above)
Comment (terjekv) [reply easybuild/easyconfigs/p/polymake/polymake-4.8-foss-2022a.eb:21]: Resolved in https://github.com/easybuilders/easybuild-easyconfigs/pull/17212/commits/ab01bd22c6e71b2c9027ca807c8e3fdb859bb240

File: easybuild/easyconfigs/p/polymake/polymake-4.8-foss-2022a.eb (hunk 3 above)
Comment (terjekv) [reply easybuild/easyconfigs/p/polymake/polymake-4.8-foss-2022a.eb:28]: Resolved in https://github.com/easybuilders/easybuild-easyconfigs/pull/17212/commits/ab01bd22c6e71b2c9027ca807c8e3fdb859bb240

File: easybuild/easyconfigs/s/SVG/SVG-2.87-foss-2022a.eb (hunk 4 above)
Comment (terjekv) [reply easybuild/easyconfigs/s/SVG/SVG-2.87-foss-2022a.eb:9]: Ah, yes, the previous version was in foss. Resolved in https://github.com/easybuilders/easybuild-easyconfigs/pull/17212/commits/ab01bd22c6e71b2c9027ca807c8e3fdb859bb240

**Comment by terjekv** [comment]
Test report by @terjekv
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
ninhursaga.uio.no - Linux RHEL 9.1 (Plow), x86_64, Intel(R) Xeon(R) Silver 4216 CPU @ 2.10GHz, 1 x NVIDIA Tesla T4, 525.60.13, Python 3.9.14
See https://gist.github.com/99dc6834aae3aa4b09c5d3b6d2e5293a for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node087.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/64f1b3a2cd729b0431cf0e5f632d1107 for a full test report.

**Comment by jfgrimm** [comment]
@boegelbot please test @ jsc-zen2

**Review by jfgrimm — APPROVED** [review]
lgtm

**Comment by jfgrimm** [comment]
Going in, thanks @terjekv!

**Pull request merged**